datasets/kz/kz_companies/crawler.py

```diff
@@ -0,0 +1,120 @@
+from time import sleep
+
+from zavod import Context
+from zavod import helpers as h
+from rigour.urls import build_url
+
+
+BASE_URL = "https://data.egov.kz/datasets/view?index=gbd_ul"
+CRAWL_URL = "https://data.egov.kz/datasets/getdata"
+CACHE_DAYS = 7
```

Comment:
We could go a bit higher here, like 30-40 days. The company registry crawlers are only run weekly, so 7 days cache makes little difference.